backend/BalanceBuddy/src/main/java/com/example/balancebuddy/config/JWTToUserConvertor.java

```diff
@@ -9,9 +9,14 @@
 import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
 import org.springframework.security.oauth2.jwt.Jwt;
 
+// Converts a JWT (JSON Web Token) into a UsernamePasswordAuthenticationToken which is designed for simple
+// presentation of a username and a password

```

Comment:
You don't need to explain in detail what a method does. You can just leave: "Converts a JWT into UsernamePasswordAuthenticationToken". Try to keep it simple and give some hints. Other devs should know how things work in detail.